tests/data/reference_graphs/original/gnmt_variable.dot

```diff
@@ -1,389 +0,0 @@
-"161 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/ModuleList[inners]/Recurrent[0]/LSTMCellForwardNNCF[cell]/__mul__" -> "162 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/ModuleList[inners]/Recurrent[0]/LSTMCellForwardNNCF[cell]/__add__";
-"162 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/ModuleList[inners]/Recurrent[0]/LSTMCellForwardNNCF[cell]/__add__" -> "163 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/ModuleList[inners]/Recurrent[0]/LSTMCellForwardNNCF[cell]/tanh";
-"162 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/ModuleList[inners]/Recurrent[0]/LSTMCellForwardNNCF[cell]/__add__" -> "168 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/cat";
-"163 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/ModuleList[inners]/Recurrent[0]/LSTMCellForwardNNCF[cell]/tanh" -> "164 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/ModuleList[inners]/Recurrent[0]/LSTMCellForwardNNCF[cell]/__mul__";
-"164 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/ModuleList[inners]/Recurrent[0]/LSTMCellForwardNNCF[cell]/__mul__" -> "165 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/ModuleList[inners]/Recurrent[0]/cat";
-"164 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/ModuleList[inners]/Recurrent[0]/LSTMCellForwardNNCF[cell]/__mul__" -> "167 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/cat";
-"165 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/ModuleList[inners]/Recurrent[0]/cat" -> "166 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/StackedRNNResetPoint/cat";
-"166 ResidualRecurrentDecoder/ModuleList[rnn_layers]/NNCF_RNN[2]/StackedRNN[rnn_impl]/StackedRNNResetPoint/cat" -> "169 ResidualRecurrentDecoder/__add__";
-"169 ResidualRecurrentDecoder/__add__" -> "170 ResidualRecurrentDecoder/Classifier[classifier]/Linear[classifier]/linear";
-"171 ResidualRecurrentDecoder/RecurrentAttention[att_rnn]/BahdanauAttention[attn]/__add__" -> "172 ResidualRecurrentDecoder/RecurrentAttention[att_rnn]/BahdanauAttention[attn]/tanh";
-"172 ResidualRecurrentDecoder/RecurrentAttention[att_rnn]/BahdanauAttention[attn]/tanh" -> "173 ResidualRecurrentDecoder/RecurrentAttention[att_rnn]/BahdanauAttention[attn]/matmul";
-"173 ResidualRecurrentDecoder/RecurrentAttention[att_rnn]/BahdanauAttention[attn]/matmul" -> "174 ResidualRecurrentDecoder/RecurrentAttention[att_rnn]/BahdanauAttention[attn]/softmax";
-"174 ResidualRecurrentDecoder/RecurrentAttention[att_rnn]/BahdanauAttention[attn]/softmax" -> "175 ResidualRecurrentDecoder/RecurrentAttention[att_rnn]/BahdanauAttention[attn]/bmm";
-}
```

Comment:
Why did this file have to be removed?

Reply:
This is an extra file. It doesn't match any test case.